src/styles/blocks/main.scss

```diff
@@ -0,0 +1,869 @@
+@use "../utils/mixins";
+
+.main {
+  background-color: #f2f6fa;
+
+  &__section {
+    margin-bottom: 120px;
+
+    @include for-tablet() {
+      margin-bottom: 148px;
+    }
+  }
+}
+
+.who-we-are {
+  padding: 160px 20px 0;
+
+  @include for-tablet() {
+    padding: 160px 41px 0;
+  }
+
+  @include for-small-desktop() {
+    padding: 148px 166px 0;
+  }
+
+  @include for-desktop() {
+    padding: 148px 300px 0;
+  }
+
+  @include the-design() {
+    padding: 148px 423px 0;
+  }
+
+  @include large-screens() {
+    padding: 148px 903px 0;
+  }
+
+  &__title {
+    color: #253757;
+    text-align: center;
+    font-size: 32px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 150%;
+    padding-bottom: 24px;
+
+    @include for-tablet() {
+      padding-bottom: 16px;
+    }
+  }
+
+  &__article {
+    color: #6c788b;
+    text-align: center;
+    font-size: 22px;
+    font-family: "Open Sans", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 150%;
+
+    @include for-tablet() {
+      font-size: 24px;
+    }
+  }
+}
+
+.our-expertise {
+  background-color: #fff;
+  border-radius: 30px;
+
+  @include for-small-desktop() {
+    margin-bottom: 148px;
+  }
+
+  &__title {
+    color: #253757;
+    text-align: center;
+    font-size: 44px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 120%;
+    padding: 72px 20px 56px;
+
+    @include for-tablet() {
+      padding: 128px 41px 64px;
+    }
+  }
+
+  &__cards {
+    @include for-small-desktop() {
+      display: flex;
+      justify-content: center;
+      gap: 167px;
+    }
+
+    @include large-screens() {
+      gap: 257px;
+    }
+  }
+
+  &__card {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    flex-direction: column;
+    align-content: center;
+    padding-bottom: 56px;
+
+    @include for-tablet() {
+      padding-bottom: 72px;
+    }
+
+    @include for-small-desktop() {
+      max-width: 215px;
+    }
+
+    &:last-child {
+      padding-bottom: 72px;
+
+      @include for-tablet() {
+        padding-bottom: 128px;
+      }
+
+      @include for-small-desktop() {
+        padding-bottom: 72px;
+      }
+    }
+  }
+
+  &__card-img {
+    width: 166px;
+    padding-bottom: 32px;
+    align-self: center;
+    transition: all 0.4s;
+
+    &:hover {
+      @include hover();
+    }
+  }
+
+  &__card-title {
+    color: #253757;
+    text-align: center;
+    font-size: 20px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 140%;
+    padding-bottom: 16px;
+  }
+
+  &__card-article {
```

Comment:
Add `max-witdh` to be aligned with the design
![image](https://github.com/mate-academy/layout_dia/assets/126275137/f0f39efe-accc-4e34-b96d-04fcbfb7c2ca)